### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
-# Changelog
+# Changelog {#Changelog}
 
-# git master {#master}
+# git master
+
+# Release 1.3.0 (2-Nov-2015)
```
Comment (tribal-tec) [inline doc/Changelog.md]: Month name now? Lunchbox etc. was number...

**Comment by tribal-tec** [comment]
barrier latency test fix should be separate commit or mentioned in the commit msg.

File: doc/Changelog.md (hunk 1 above)
Comment (eile) [inline doc/Changelog.md]: Others where month... not sure which one to converge to. Name is more internationally friendly.

**Comment by eile** [comment]
done.

**Comment by tribal-tec** [comment]
+1

**Pull request merged**